Frontend/CO.CDP.OrganisationApp/Pages/Organisation/SupplierToBuyerOrganisationDetailsSummary.cshtml

```diff
@@ -0,0 +1,87 @@
+@page "/organisation/{id}/supplier-to-buyer-organisation-details-summary"
+@using CO.CDP.OrganisationApp.Constants
+@model CO.CDP.OrganisationApp.Pages.Organisation.SupplierToBuyerOrganisationDetailsSummaryModel
+@{
+    ViewData["Title"] = "Check your answers";
+    var details = Model.SupplierToBuyerDetailsModel!;
+
+    var supplierToBuyerOrganisationTypePage = $"/organisation/{@Model.Id}/supplier-to-buyer-organisation-type";
+    var supplierToBuyerDevolvedRegulationsPage = $"/organisation/{@Model.Id}/supplier-to-buyer-devolved-regulation";
+    var supplierToBuyerSelectDevolvedRegulationsPage = $"/organisation/{@Model.Id}/supplier-to-buyer-select-devolved-regulation";
+
+    var backLink = $"/organisation/{@Model.Id}/supplier-to-buyer-select-devolved-regulation";
+
+    if (details.Devolved == true)
+    {
+        backLink = $"/organisation/{@Model.Id}/supplier-to-buyer-select-devolved-regulation";
+    }   
+}
+
+<a href="@backLink" class="govuk-back-link">
+    Back
+</a>
+
+<main class="govuk-main-wrapper" id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <partial name="_ErrorSummary" model="@ModelState" />
+            <span class="govuk-caption-l">Buyer details</span>
```

Comment:
Caption should go inside of the h1, before the title

Reply:
@andymantell the caption is moved inside the H1 now.